### PR TITLE
Fail2ban documentation

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/5489

### CODE REVIEW

**Comment by stephdl** [comment]
@gsanchietti I cannot add you as reviewer, what do you think ?

**Comment by stephdl** [comment]
@gsanchietti it should be good, waiting your input

File: administrator-manual/en/fail2ban.rst [hunk 1]
```diff
@@ -0,0 +1,160 @@
+========
+Fail2ban
+========
+
+Fail2ban scans log files (e.g. /var/log/apache/error_log) and bans IPs that show the malicious signs – too many password failures, seeking for exploits, etc. Generally Fail2Ban is then used to update firewall rules to reject the IP addresses for a specified amount of time, although any arbitrary other action (e.g. sending an email) could also be configured. Out of the box Fail2Ban comes with filters for various services (apache, courier, ssh, etc).
```
Comment (gsanchietti) [inline administrator-manual/en/fail2ban.rst]: /var/log/apache/error_log should be marked as :file:`/var/log/apache/error_log`
See https://github.com/NethServer/docs#editing-guidelines

Also use the correct case for:
Apache, Courier, SSH

File: administrator-manual/en/fail2ban.rst [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+Fail2Ban is able to reduce the rate of incorrect authentications attempts however it cannot eliminate the risk that weak authentication presents. Configure services to use only two factors or public/private authentication mechanisms if you really want to protect services.
```
Comment (gsanchietti) [inline administrator-manual/en/fail2ban.rst]: Add a comma after the word "attempts".

Also, I'd change this:
"Configure services to use only two factors or public/private authentication mechanisms if you really want to protect services." 
To something like:
"To improve the security, open the access to service only for secure networks using the firewall."

File: administrator-manual/en/fail2ban.rst [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+Install from the Software Center or use the command line: ::
+
+  yum install nethserver-fail2ban
+
+
+Settings
+========
+
+Fail2ban is configurable in the security category of the server-manager. Most of settings can be changed in the ``Configuration`` tab, only really advanced settings must be configured by the terminal. The ``Ban status`` tab displays the statistic and the banned IPs.
```
Comment (gsanchietti) [inline administrator-manual/en/fail2ban.rst]: Again, use `guilabel` to mark name of pages and labels
```
:guilabel:`xxxxx`
```

File: administrator-manual/en/fail2ban.rst [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+You can also activate changes and restart the fail2ban service by: ::
```
Comment (gsanchietti) [inline administrator-manual/en/fail2ban.rst]: Move this to README for developer manual.

File: administrator-manual/en/fail2ban.rst [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+  signal-event nethserver-fail2ban-save
+
+Jails
+-----
+
+A jail is enabled and start to protect a service when you install a new module, the relevant jail (if existing) is automatically activated by the event 'runlevel-adjust' which is launched at the end of the yum process.
```
Comment (gsanchietti) [inline administrator-manual/en/fail2ban.rst]: The user doesn't know what is the runlevel-adjust.
I'd change it  to something like:
"the relevant jail is automatically activated then new packages are installed"

File: administrator-manual/en/fail2ban.rst [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+Network
+-------
+
+Allow bans on the LAN
+    By default the failed attempts from your Local Network are ignored, except when you enabled the option.
+
+
+IP Whitelisting
+    Ip listed in the text area will be never banned by fail2ban (one IP per line).
```
Comment (gsanchietti) [inline administrator-manual/en/fail2ban.rst]: "Ip" should always be written "IP"

File: administrator-manual/en/fail2ban.rst [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+    Enable to send administrative emails.
+
+Administrators emails
+    List of email addresses of administrators (one address per line).
+
+Notify jail start/stop events
+    Send email notifications when a jail is started or stopped.
+
+
+E-smith Database
```
Comment (gsanchietti) [inline administrator-manual/en/fail2ban.rst]: Move "E-smith Database" chapter inside the README.rst for the developer manual.

**Pull request merged**